frontend/public/locales/en/translation.json

```diff
@@ -1305,8 +1307,8 @@
     "infraEnv.bulkAction.createInfraEnv": "Create infrastructure environment",
     "infraEnv.createCluster": "Create infrastructure environment",
     "infraEnv.delete.plural": "Delete infrastructure environments",
-    "infraEnv.emptyStateBody": "To create a pool of hosts that is ready for cluster creation, create an infrastructure environment. Once a new infrastructure environment is created, you can discover, provision, and add hosts to it.",
-    "infraEnv.emptyStateHeader": "No infrastructure environments yet",
+    "infraEnv.emptyStateBody": "Click the <bold>Create infrastructure environment</bold> button to create an infrastructure environment. Once a new infrastructure environment is created, you can discover, provision, and add hosts to it.",
```

Comment:
I think "create your resource" would be less redundant here.

Reply:
"Click <bold>Create infrastructure environment</bold> to create your resource.  You can discover, provision, and add hosts to the infrastructure environment after creation."

@swopebe

Reply:
Perfect @RobDolares